package.json

```diff
@@ -32,6 +32,9 @@
     "type-fest": "^0.20.2",
     "yargs": "^17.0.2"
   },
+  "resolutions": {
+    "**/eslint-plugin-deprecation": "1.2.1"
+  },
```

Comment:
@jfaltermeier rather than add a resolution, why do we not simply update the dependency to use `1.2.1` directly?

https://github.com/eclipse-theia/theia-blueprint/blob/f6e2116b802b18b4dfee756ef639b814362b5d20/package.json#L24

to:

```json
"eslint-plugin-deprecation": "1.2.1"
```

Reply:
So usually I like resolution for issues like this because it also works for transient dependencies. I started with restricting `typescript-eslint/experimental-utils` which led me to `eslint-plugin-deprecation`. 
But in this case we only have a direct dependency, so I'll update this